geth/account/accounts_test.go

```diff
@@ -117,17 +407,394 @@ func (s *AccountsTestSuite) TestVerifyAccountPassword() {
 // TestVerifyAccountPasswordWithAccountBeforeEIP55 verifies if VerifyAccountPassword
 // can handle accounts before introduction of EIP55.
 func (s *AccountsTestSuite) TestVerifyAccountPasswordWithAccountBeforeEIP55() {
-	keyStoreDir, err := ioutil.TempDir("", "status-accounts-test")
-	s.NoError(err)
-	defer os.RemoveAll(keyStoreDir)
+	require := s.Require()
+
+	// init empty keyStore
+	emptyKeyStoreDir, err := ioutil.TempDir(os.TempDir(), "status-empty-keystore")
+	require.NoError(err)
+	require.NotEmpty(emptyKeyStoreDir)
+	defer os.RemoveAll(emptyKeyStoreDir)
 
-	// Import keys and make sure one was created before EIP55 introduction.
-	err = common.ImportTestAccount(keyStoreDir, "test-account1-before-eip55.pk")
-	s.NoError(err)
+	// import keys and make sure one was created before EIP55 introduction.
+	err = common.ImportTestAccount(emptyKeyStoreDir, "test-account1-before-eip55.pk")
+	require.NoError(err)
 
-	acctManager := account.NewManager(nil)
+	// account manager
+	accountManager := account.NewManager(nil)
 
+	// test
 	address := gethcommon.HexToAddress(TestConfig.Account1.Address)
-	_, err = acctManager.VerifyAccountPassword(keyStoreDir, address.Hex(), TestConfig.Account1.Password)
-	s.NoError(err)
+	_, err = accountManager.VerifyAccountPassword(emptyKeyStoreDir, address.Hex(), TestConfig.Account1.Password)
+	require.NoError(err)
+}
+
+/*
+
+func (s *AccountsTestSuite) TestSelectAccount() {
```

Comment:
Are these tests to be removed?

Reply:
still in progress: reviewing them one by one :)